content/version_control/VCS.tex

```diff
@@ -0,0 +1,322 @@
+  % You might wish to add the option [pausesections]
+\end{frame}
+
+% Section and subsections will appear in the presentation overview
+% and table of contents.
+%\section{First Main Section}
+%
+%\subsection{First Subsection}
+
+\begin{frame}{What is it?}%{Optional Subtitle}
+\section{What is it?}
+  \begin{itemize}
+  \item {
+    A method to store data/files in a central location.
+  }
+  \item {
+    A backup when things go wrong.
+  }
+  \end{itemize}
+\end{frame}
+
+%\subsection{}
+
+% You can reveal the parts of a slide one at a time
+% with the \pause command:
+\begin{frame}{Why should we use it ?}
+\section{Why should we use it ?}
+  \begin{itemize}
+  \item {
+    A complete long-term change history of every file.
+%    \pause % The slide will pause after showing the first item
+  }
+  \item {   
+    Branching and merging
+  }
+  \item {   
+    Ability to trace each change made
+  }
+  \item {   
+	Allows multiple developers to work simultaneously
+  }
+  % You can also specify when the content should appear
+  % by using <n->:
+%  \item<3-> {
+%    Third item.
+%  }
+%  \item<4-> {
+%    Fourth item.
+%  }
+  % or you can use the \uncover command to reveal general
+  % content (not just \items):
+%  \item<5-> {
+%    Fifth item. \uncover<6->{Extra text in the fifth item.}
+%  }
+  \end{itemize}
+\end{frame}
+
+\begin{frame}{Types of VCS}{Centralized VCS}
+\section{Types of VCS}
+\subsection{Centralized VCS}
+  \begin{columns}
+\column{0.5\textwidth}
+\begin{itemize}
+  \item {
+    Centralized VCS keeps the history of changes on a central server.
+  }
+  \item {
+    Designed with the intent that there is only one true source  
+  }
+  \item {
+    If you want to make a copy of your data, you have to copy/paste it, literally.  
+  }
+  \item{
+	Sourceforge.net uses this type of versioning.  
+  }
+  \item{
+      Example: SVN (Subversion)
+  }  
+  \end{itemize}
+\column{0.5\textwidth}
+\begin{figure}
+	 \includegraphics[width=.9\textwidth]{cvs}
+	 \caption{Centralized Version System \cite{website}}
+ \end{figure}
+\end{columns}
+\end{frame}	
+
+\begin{frame}{Types of VCS}{Decentralized VCS}
+\subsection{Distributed VCS}
+\begin{columns}
+\column{0.5\textwidth}
+\begin{itemize}
+		  \item {
+		    In distributed VCS, everyone has a local copy of the entire work’s history
+		  }
+		  \item {
+		    Each repository is as good as the other
+		  }
+		  \item{
+			Prone to central server crash
+		  }
+		  \item{
+			Your local copy is a repository, and you can commit to it and get all benefits of source control - Offline Source Control.
+		  }	  
+		  \item{
+			Mozilla Firefox uses this kind of versioning.  
+		  }
+		  \item{
+            Example: Git		  
+		  }
+	  \end{itemize}
+\column{0.5\textwidth}
+\begin{figure}
+	 \includegraphics[width=.9\textwidth]{dvs}
+	 \caption{Distributed Version System \cite{website}}
+ \end{figure}
+\end{columns}
+\end{frame}	
+
+\begin{frame}{Types of VCS}{Git vs SVN}
+\subsection{Git vs SVN}
```

Comment:
The message here is a little confusing. Group this into pros and cons of git and SVN so they are a little bit more clear. 
In the end, the message the students should get is "Why do I need to use git?"